insights/parsers/proc_limits.py

```diff
@@ -62,9 +76,16 @@ def __iter__(self):
 
     def parse_content(self, content):
         self.data = []
-        self.max_processes = ''
         if 'No such file or directory' not in content[0]:
             self.data = parse_fixed_table(content, header_substitute=HEADER_SUBSTITUTE)
         for row in self.data:
-            if 'Max processes' == row['Limit']:
-                self.max_processes = row['Hard_Limit']
+            row['Limit'] = row['Limit'].lower().replace(' ', '_')
+            setattr(self, row['Limit'], row['Hard_Limit'])
```

Comment:
Is the *Units* column important here, for example if *Hard Limit* is 4096, would a user of the value need to know the units?

Reply:
It depends. I will add it together with the soft_limit. Thanks.